.ebextensions/change_upload_size.config

```diff
@@ -0,0 +1,8 @@
+files:
+  "/etc/php.d/project.ini" :
```

Comment:
Doesn't this do everything necessary? With this in place, why do we need `.platform/files/php.ini`?